pkg/prediction/config/config.go

```diff
@@ -19,9 +19,9 @@ const (
 
 	// following is node exporter metric for node cpu/memory usage
 	// NodeCpuUsagePromQLFmtStr is used to query node cpu usage by promql,  param is node name which prometheus scrape, duration str
-	NodeCpuUsagePromQLFmtStr = `sum(count(node_cpu_seconds_total{mode="idle",instance=~"%s.*"}) by (mode, cpu)) - sum(irate(node_cpu_seconds_total{mode="idle",instance=~"%s.*"}[%s]))`
+	NodeCpuUsagePromQLFmtStr = `sum(count(node_cpu_seconds_total{mode="idle",instance="%s"}) by (mode, cpu)) - sum(irate(node_cpu_seconds_total{mode="idle",instance="%s"}[%s]))`
 	// NodeMemUsagePromQLFmtStr is used to query node cpu memory by promql,  param is node name, node name which prometheus scrape
-	NodeMemUsagePromQLFmtStr = `sum(node_memory_MemTotal_bytes{instance=~"^%s.*"} - node_memory_MemAvailable_bytes{instance=~"^%s.*"})`
+	NodeMemUsagePromQLFmtStr = `sum(node_memory_MemTotal_bytes{instance="%s"} - node_memory_MemAvailable_bytes{instance="%s"})`
```

Comment:
This regex is used to match some instance including ip:port style. And it depends on your monitoring system, so we do not assume the instance is nodename or ip only.  

If we modify this prom template, we need find a more general way.

<img width="956" alt="image" src="https://user-images.githubusercontent.com/8846958/156512460-a6fe7d2f-6af3-4372-b730-e73beb0cf9d1.png">

Reply:
I know, so I pushed a new commit that is used to match some instance including ip:port style.
![image](https://user-images.githubusercontent.com/29157728/156522235-f1d63b78-29f2-44a5-ab7f-453f860565ca.png)

Reply:
\LGTM